lib/localenv/clusterenv.go

```diff
@@ -54,7 +68,12 @@ type ClusterEnvironment struct {
 
 // NewClusterEnvironment initializes local cluster services and
 // returns a new instance of cluster environment
+// FIXME: this is deprecated. Use localenv.LocalEnvronment.NewClusterEnvironment instead
```

Comment:
Why is this deprecated? Can there be no scenario where you want to create a cluster environment without having a local env?